res/mixxx.qrc

```diff
@@ -29,6 +29,7 @@
         <file>images/library/ic_library_traktor.svg</file>
         <file>images/library/ic_library_rekordbox.svg</file>
         <file>images/library/ic_library_serato.svg</file>
+        <file>images/library/ic_library_tracksuggestion.svg</file>
```

Comment:
How about using a light bulb here? https://icon-sets.iconify.design/mdi/lightbulb-on/
Maybe with an exclamation mark or surrounded by a speech bubble.

Reply:
Oh, that's a good icon. It's shows like the Mixxx has an idea to show suggestions.